WordPress/gutenberg
# JSON Schema Reorganization and Fixes

## What?


Post-update code organization for JSON schema.

JSON structure changes:
- Inline `useRootPaddingAwareAlignments` property.
- Inline `fontFace` items.
- Sort `settingsProperties` alphabetically.
- Ensure every subschema with `properties` has `"type": "object"`.
- Ensure every subschema with `propertyNames` has `"type": "object"`.
- Alphabetize `stylesProperties`.

Fixes:
- Remove references to WordPress versions.
- Add missing ref support for `textAlign` and `textColumns`.
- Reuse `#/definitions/stylesElementsPseudoSelectorsProperties`

## Why?


Follow-up to #62462 to fix some things found while working on it.

## How?


See the commits for details.

## Testing Instructions





Check that [the new docs](https://github.com/WordPress/gutenberg/blob/update/json-schema-cleanup/docs/reference-guides/theme-json-reference/theme-json-living.md) and [new schema](https://github.com/WordPress/gutenberg/blob/update/json-schema-cleanup/schemas/json/theme.json) look correct.

Do all of the properties seem like they're in an intuitive order that makes sense?

Is the JSON file readable? Would you more easily be able to figure out where to add a new property if you need to?

### Testing Instructions for Keyboard


N/A

## Screenshots or screencast 

N/A
